capstone/backend/src/test/java/com/google/servlets/GetItemTypesServletTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.servlets;
+
+import org.mockito.Mockito;
+import java.io.IOException;
+import java.util.HashMap;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletResponse;
+import junit.framework.TestCase;
+
+public class GetItemTypesServletTest extends TestCase {
+  private static final String PAGE_KEY = "page";
+  private static final String PAGE = "0";
+
+  public void testGetItems() throws ServletException, IOException {
+    HashMap<String, String> map = new HashMap<>();
+    map.put(PAGE_KEY, PAGE);
+    SetupObj setupObj = ServletTestUtil.setupMockDataGet(map);
+
+    GetItemTypesServlet servlet = new GetItemTypesServlet();
```

Comment:
Mock the `LibraryFunctions.getItemTypes` here to return some fake data when the function is called, so you don't call the real database in the test. It'll also be more clear where the "CEREAL, MILK, WATER" are coming from. (also if we do change the data in the database, this test may become flaky, which is not great)

Reply:
so unfortunately you cannot mock a static method without using powermock which is kinda weird and i'm having trouble figuring out the dependency stuff, in the meantime I built a new database called test-db for us to use for all of our testing, this is something we can talk about but at the very least we are no longer using the prod database

Reply:
Okay, thanks for looking into it! You could do something like:
in your servlet add a function called 
```
getItemTypes(){
  return LibraryFunctions.getItemTypes();
}
```
Then in your test, 
```
Mockito.when(GetItemTypesServlet.getItemTypes()).thenReturn("Some fake data");
```
But this is fine for now.